code/frontend/src/app/features/seeker-stats/quality-tab/quality-tab.component.scss

```
@@ -280,3 +264,84 @@
+}
+
+// Table cell content
+.cell-title {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-1);
+
+  &__text {
+    font-weight: 500;
+    color: var(--text-primary);
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)